test/e2e/snaps/test-snap-bip32.spec.js

```diff
@@ -0,0 +1,139 @@
+const { strict: assert } = require('assert');
```

Comment:
Looks to be some linting errors. Should be fixable with `yarn lint:fix`?

Reply:
I think i have dispatched all the linting errors. There's still a warning but I don't think that will fail and it isn't in anything i touched with this PR